react-components/table-intro/step3.md

```diff
@@ -13,3 +13,5 @@ Apply this transformation to all of the column definitions.
 <strong>Note: </strong> The table should appear exactly as it did before:
```

Comment:
I added some more information to this step to make it more obvious what the user is doing and why. Hope that helps, let me know what you think.

We separated the transform of column format and row format into multiple steps because transforming the rows is a bit more lengthy to explain, figured by having people do the exercise with the simpler columns first, the next step wouldn't be such a shock. WDYT?

Reply:
I didn't notice the "Show solution" button at the bottom, but that would be fine.